### DISCUSSION TITLE
Add an opportunity to assign OWASP Risk rating to the vulnerability

### DISCUSSION THREAD

**Discussion by gano3s** [reconmap/reconmap#84] [Ideas]
It would be great to have an opportunity to add OWASP Risk rating to the vulnerability. Especially for penetration tests of the web applications. https://owasp.org/www-community/OWASP_Risk_Rating_Methodology and accordingly be able to sort the discovered issues in generated report based on this risk rating.

An example of the Risk Rating calculator is here https://javierolmedo.github.io/OWASP-Calculator/.

**Comment by santiagolizardo** [comment]
Great idea. I will add an option to decide which risk rating use per project. Initial options will be CVSS and OWASP Risk Rating, but others could be added in the future.
Calculators will be added to rating fields as well to ease the generation of the scores.
Thanks for the suggestion!
